x-pack/plugins/uptime/public/components/common/header/action_menu.tsx

```diff
@@ -6,22 +6,60 @@
  */
 
 import React from 'react';
-import { EuiButtonEmpty, EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+import { EuiButtonEmpty, EuiFlexGroup, EuiFlexItem, EuiToolTip } from '@elastic/eui';
 import { i18n } from '@kbn/i18n';
 import { useKibana } from '../../../../../../../src/plugins/kibana_react/public';
-import { HeaderMenuPortal } from '../../../../../observability/public';
+import {
+  createExploratoryViewUrl,
+  HeaderMenuPortal,
+  SeriesUrl,
+} from '../../../../../observability/public';
 import { AppMountParameters } from '../../../../../../../src/core/public';
+import { useUptimeSettingsContext } from '../../../contexts/uptime_settings_context';
+import { useGetUrlParams } from '../../../hooks';
 
 const ADD_DATA_LABEL = i18n.translate('xpack.uptime.addDataButtonLabel', {
   defaultMessage: 'Add data',
 });
 
+const ANALYZE_DATA = i18n.translate('xpack.uptime.analyzeDataButtonLabel', {
+  defaultMessage: 'Analyze data',
+});
+
+const ANALYZE_MESSAGE = i18n.translate('xpack.uptime.analyzeDataButtonLabel.message', {
+  defaultMessage:
+    'EXPERIMENTAL - Analyze Data allows to select and filter result data in any dimensions and look for the cause or impact of performance problems.',
```

Comment:
nit typo: `'EXPERIMENTAL - Analyze Data allows you to select and filter result data in any dimension and look for the cause or impact of performance problems.'`